src/status_im/profile/screen.cljs

```diff
@@ -113,6 +114,13 @@
             [status-view {:style  (st/status-text (:height (r/state component)))
                           :status status}])])})))
 
+(defn- navigate-to-phone-change
+  "Switch user to the console issuing the !phone command automatically to let him change his phone number."
+  []
+  (dispatch [:navigate-to :chat console-chat-id])
+  (dispatch [:set-chat-command :phone])
+  )
```

Comment:
IMO this is acceptable, but it is better to move it to another handler just for clarity and readability